scripts/prereqs/install_prereqs.sh

```diff
@@ -59,7 +59,7 @@ fi
 ################################# Should not have to change anything below ############################
 
 CENTOS_VERSION=0
-PARENT_DIR="$(dirname $(python -c "import os; import sys; print(os.path.realpath(sys.argv[1]))" $0))"
+PARENT_DIR="$(dirname $0)"
```

Comment:
Are you assured that $0 will always be the realpath?

Reply:
No, but I don't think we need it be. `PARENT_DIR` only get used in three  places, including the obsolete protobuf install function. In each of those cases, I think the script works fine if it gets back an absolute path or a relative path for `PARENT_DIR`.